tests/integration_tests/validation/conformance_suite_configurations/cipc_current.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import PurePath, Path
+from tests.integration_tests.validation.conformance_suite_config import ConformanceSuiteConfig, ConformanceSuiteAssetConfig
+
+config = ConformanceSuiteConfig(
+    args=[
+        '--disclosureSystem', 'cipc',
+    ],
+    assets=[
+        ConformanceSuiteAssetConfig.local_conformance_suite(
+            Path('cipc'),
+            entry_point=Path('index.xml'),
+        ),
+    ],
+    info_url='https://www.cipc.co.za/?page_id=4400',
+    name=PurePath(__file__).stem,
+    network_or_cache_required=False,
```

Comment:
I think we do require the cache here since we don't configure a taxonomy package. Although, taxonomy packages do appear to exist for CIPC (referenced [here](https://taxonomies.xbrl.org/api/v0/taxonomy)). 
2023: https://www.cipc.co.za/wp-content/uploads/2023/08/cipc_2023-09-07.zip
2019: http://xbrl.cipc.co.za/Other_Taxonomy_Files/cipc_2019-09-30.zip (appears to have moved/be private)